flybywiresim/a32nx
# fix: only update ewd memo lines on change

Fixes #[issue_no]

## Summary of Changes

Adds a check for the pfwc memos to make sure we only send updates when the messages actually change. This might improve performance and prevent  the memos from disappearing for a short moment (as we clear the messages on every update as well).

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

 - Make sure memos on right and left side of the EWD still appear and become updated correctly (check T.O. Memos and LDG memos as an example for the left messages. Parking Brake and Landing lights for the right messages).
 - Check failures appear and can be cleared.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
